xml/install_caasp_heat_templates.xml

```diff
@@ -571,6 +584,12 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
         Log in to each node and add the update repository.
        </para>
        <screen>&prompt.ardana;sudo zypper ar http://nu.novell.com/SUSE/Updates/SUSE-CAASP/3.0/x86_64/update/ caasp_update</screen>
+       <para>
+        <literal>nu.novell.com</literal> must be reachable in order to proceed
+        with the update, which requires a valid DNS server having been configured.
```

Comment:
Sorry, the "which" here  doesn't make sense. My suggestion is to use "ensure a valid DNS server has been configured." or something similar :)